### ISSUE TITLE
Resolve discrepancy with number of compounds discovered via maven_core and maven gui for TGs

### ISSUE THREAD

**Issue by PMSeitzer** [eugenemel/maven#234]
```
lib_search_params <- list(
  "consensusMinFractionMs2Scans"=0.25,
  "ms2MinNumMatches"=2,
  "ms2MinNumUniqueMatches"=1,
  "spectralCompositionAlgorithm"="AUTO_SUMMARIZED_ACYL_CHAINS_SUM_COMPOSITION",
  "ms1IsFindPrecursorIon"=TRUE,
  "ms1PpmTolr"=3,
  "ms1ScanFilter"="SIM"
)
```

**Comment by PMSeitzer** [comment]
![Issue234-TG-search-params](https://user-images.githubusercontent.com/1757701/87840709-eea83d80-c855-11ea-8a97-b4c860d53ccc.png)

![Issue234-libraries](https://user-images.githubusercontent.com/1757701/87840738-0f709300-c856-11ea-8ef2-477bca1b552c.png)

![Issue234-maven-gui](https://user-images.githubusercontent.com/1757701/87840741-11d2ed00-c856-11ea-884f-f4f4c30a48b2.png)

**Comment by PMSeitzer** [comment]
running `mzkitcpp::DI_pipeline()` yields the following:
>mzkitcpp::DI_pipeline(): Identified 63 compounds.

in the saved `mzrollDB`, 47 compounds

also reading the R results,
```
> ms2_results$compoundName %>% unique() %>% length()
[1] 61
```

Need to resolve all of these discrepancies

**Comment by PMSeitzer** [comment]
Also see associated `mass_spec` issue:
https://github.com/calico/mass_spec/issues/434

**Comment by PMSeitzer** [comment]
![MAVEN_TG_36_1_screenshot](https://user-images.githubusercontent.com/1757701/88211351-769d9700-cc0a-11ea-96c0-28f52ce194cb.png)
![mzkitcpp_TG_36_1_screenshot](https://user-images.githubusercontent.com/1757701/88211361-78fff100-cc0a-11ea-8015-d57c89c1abe7.png)

**Comment by PMSeitzer** [comment]
This was resolved with the completion of #238
